pkg/controller/appsync/appsync.go

```diff
@@ -16,16 +16,22 @@
 package appsync
 
 import (
-	"context"
+	"encoding/json"
 	"errors"
+	"fmt"
+	"io"
 	"net/http"
 
+	"github.com/google/exposure-notifications-server/pkg/logging"
 	"github.com/google/exposure-notifications-verification-server/pkg/config"
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
 	"github.com/google/exposure-notifications-verification-server/pkg/render"
+	"github.com/hashicorp/go-multierror"
 )
 
+const playStoreLink = `https://play.google.com/store/apps/details?id=`
```

Comment:
are there international play store links that we need to consider?
@flagxor

Reply:
Play store will use device locale so I think this should be the same regardless of locale.